app/models/invite_redeemer.rb

```diff
@@ -36,6 +36,12 @@ def self.create_user_from_invite(invite, username, name, password=nil)
       user.password = password
     end
 
+    if !SiteSetting.must_approve_users? || (SiteSetting.must_approve_users? && invite.invited_by.staff?)
+      user.approved = true
+      user.approved_by_id = invite.invited_by_id
+      user.approved_at = Time.now
```

Comment:
`Time.zone.now` is preferred in Rails project. This is a good read as well :) http://danilenko.org/2012/7/6/rails_timezones/